ambry-rest/src/main/java/com/github/ambry/rest/PublicAccessLogHandler.java

```diff
@@ -100,6 +106,7 @@ public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise)
         logMessage.append(", ");
         logMessage.append("status=").append(response.status().code());
         logMessage.append(", ");
+        structuredLogMessage.put("status", String.valueOf(response.status().code()));
```

Comment:
We also log information related to received request such as remote address, headers, etc in `channelRead()` method. May be we would need those information too in structured log message?

Reply:
Thanks, good catch, I added those.